src/main/java/net/pms/PMS.java

```diff
@@ -582,26 +565,28 @@ public void configurationChanged(ConfigurationEvent event) {
 		OutputParams outputParams = new OutputParams(configuration);
 
 		// Prevent unwanted GUI buffer artifacts (and runaway timers)
-		outputParams.hidebuffer = true;
+		outputParams.setHidebuffer(true);
```

Comment:
@valib Can we revert the changes in this file before this line? It looks like some automatic formatting happened which is fine but it would be easier for me if we did formatting after we are in sync with DMS because it creates more conflicts for me

Reply:
If you want I can do that reverting